.github/workflows/tests_extra.yml

```
@@ -20,5 +20,7 @@ jobs:
     if: (github.repository == 'spacetelescope/roman_datamodels' && (github.event_name == 'schedule' || github.event_name == 'push' || github.event_name == 'workflow_dispatch' || contains(github.event.pull_request.labels.*.name, 'Weekly CI')))
     with:
       envs: |
+        - macos: py311-devdeps
+        - macos: py312-devdeps
```

```suggestion
        - macos: py311
        - macos: py312
```
No need to use devdeps for these.